luigi/static/visualiser/js/visualiserApp.js

```diff
@@ -20,6 +20,15 @@ function visualiserApp(luigi) {
     };
     var VISTYPE_DEFAULT = 'svg';
 
+    function escapeHtml(unsafe) {
```

Comment:
why not have this function in a separate file and imported from both places?

Reply:
The introduced complexity is not worth to me. I don't really know what JS standard the code base uses.

Reply:
OK I used the old ugly way to address this.